app.json

```diff
@@ -29,7 +29,8 @@
       "infoPlist": {
         "NSCameraUsageDescription": "Diese App nutzt die Kamera um QR-Codes zu scannen.",
         "NSPhotoLibraryUsageDescription": "Diese App nutzt die Medienbibliothek für die Auswahl eines Profilbildes."
-      }
+      },
+      "usesIcloudStorage": true
```

Comment:
i am thinking about moving this to particular releases and not in the main branch with 2.6.0, because not every city will have this feature.

Reply:
c083613aba07130fb5d2e455f18e02e0f2c0b5cd